ang/test/civicase/ActivitiesCalendar.spec.js

```diff
@@ -592,6 +624,65 @@
       });
     });
 
+    describe('"see more" link url', function () {
+      var currentRouteParams, queryParams, url;
+
+      beforeEach(function () {
+        currentRouteParams = {
+          dtab: 0,
+          foo: 'foo',
+          af: { bar: 'bar' }
+        };
+
+        initController(null, {
+          '$route': { current: { params: currentRouteParams } }
+        });
+
+        url = $scope.seeMoreLinkUrl(dates.yesterday);
+        queryParams = extractQueryStringParams();
+      });
+
+      it('is a trusted url', function () {
+        expect(url.$$unwrapTrustedValue).toBeDefined();
+      });
+
+      it('redirects to the activities feed tab', function () {
+        expect(queryParams.dtab).toBe('1');
+      });
+
+      it('opens by default the "filter activites" section', function () {
+        expect(queryParams.af['@moreFilters']).toBe(true);
+      });
+
+      it('automatically filters the feed by the given date', function () {
+        expect(queryParams.af.activity_date_time).toEqual({
+          'BETWEEN': [
+            moment(dates.yesterday).startOf('day').format('YYYY-MM-DD+HH:mm:ss'),
+            moment(dates.yesterday).endOf('day').format('YYYY-MM-DD+HH:mm:ss')
+          ]
+        });
+      });
+
+      it('keeps the rest of the params of the current route', function () {
+        expect(queryParams.foo).toBe(currentRouteParams.foo);
+        expect(queryParams.af.bar).toBe(currentRouteParams.af.bar);
+      });
+
+      function extractQueryStringParams () {
```

Comment:
I rather get rid of this function and try to use something provided by Angular. But if we are going to keep it, it's better to document it.